src/timeatlas/time_series/time_series.py

```diff
@@ -201,6 +202,18 @@ def prepend(self, ts: 'TimeSeries'):
         new_series = concat([ts.series, self.series])
         return TimeSeries(new_series, self.metadata)
 
+    def chunkify(self, n: int) -> List['TimeSeries']:
+        """
+        Splits TimeSeries into chunks of length n
+
+        :param n: length of chunks
+        :return: List of TimeSeries
+        """
+
+        ts_chunks = [TimeSeries(v) for n, v in self.series.groupby(np.arange(len(self.series)) // n)]
```

Comment:
The metadata object isn't propagated in the chunks, use the TimeSeries constructor second argument to fix that.